The header contains vectors of long long, which are only available with the
VSX instruction set. We don't compile with VSX on big endian systems.

--- nss/lib/freebl/gcm.h
+++ nss/lib/freebl/gcm.h
@@ -30,7 +30,7 @@
 #include <arm_neon.h>
 #endif
 
-#ifdef __powerpc64__
+#if defined(__powerpc64__) && defined(__VSX__)
 #include "altivec-types.h"
 
 /* The ghash freebl test tries to use this in C++, and gcc defines conflict. */
